### ISSUE TITLE
slicing problem with tree supports

### ISSUE THREAD

**Issue by Donald100hi** [Ultimaker/Cura#18320]
### Cura Version

5.60

### Operating System

windows 11

### Printer

ender 3

### Name abnormal settings

tree support 

### Describe model location

edge

### Describe your model

yes,no,no

### Add your .zip here ⬇️

[CE3E3V2_Fabulous Jofo-Jaagub (13).zip](https://github.com/Ultimaker/Cura/files/14275686/CE3E3V2_Fabulous.Jofo-Jaagub.13.zip)

it does work after slicing multiple times

**Comment by WhatNot911** [comment]
Yup it's a common occurence. Apparently they releases a 5.7.0beta or something with extended logging for this problem.
